src/toolkits/drawing_classifier/dc_data_iterator.hpp

```diff
@@ -38,10 +38,10 @@ class data_iterator {
      *
      * If empty, then the output will not contain labels or weights.
      */
-    std::string target_column_name;
+    std::string target_column_name{"target"};
 
     /** The name of the feature column. */
-    std::string feature_column_name;
+    std::string feature_column_name{"feature"};
```

Comment:
Why do we need to provide default here? Shouldn't this always be supplied by the client?

Reply:
This is purely for debugging purposes. The dc_iterator will throw exceptions with msg containing empty column names (default value), which is not informational for locating the error. Ideally, users will always override the value.